### ISSUE TITLE
SteadyMed

### ISSUE THREAD

**Issue by grigs** [cloudfour/pwastats#140]
> **Note:** This template is for new case study content submissions only. 
Feel free to replace the outline below if you are submitting a general issue instead.

---

### Summary

 For instance, in a comparison of December 2016 to December 2017, traffic to the SteadyMed site saw a 54% increase in the number of sessions.

### Source

https://blog.pint.com/2018/02/13/steadymed-therapeutics-progressive-web-app/
### Tags

- Offline
- Service Worker
- _needs more tags_

_List all that apply from the [tags directory](https://github.com/cloudfour/pwastats/tree/master/_tags)._

**Comment by derekshirk** [comment]
I'd like to call into question the inclusion of this site based on the source info (and not just because it a pharmaceutical company 😉) .

In the [source URL](https://blog.pint.com/2018/02/13/steadymed-therapeutics-progressive-web-app) the article references several aspects of the redesign including: 

- Full-Screen Navigation
- Tiled/Modular Layout
- _Some_ PWA "Methodology"
- Use of SVGs
- Use of WebP image format

**PWA information**

> SteadyMed’s new website was developed as a Progressive Web App (PWA) to optimize performance, increase accessibility, and support offline capability. While building the site, PINT applied best practice methodologies in order to achieve these goals.

While it does mention offline capability, the specifics of the PWA work receives very little detail. 

In summary, It feels like a stretch to claim that the "54% increase in the number of sessions" was directly related to the PWA efforts or a culmination of the other changes, or the release of the new drug in 2017 that the article mentions<sup>1<sup/>. 

In closing I think that more information would be needed in order to consider including this on pwastats.com

---

<sup>1</sup> The NDA submission for their investigational drug product, Trevyent, in June 2017

**Comment by grigs** [comment]
I can no longer find a service worker or a manifest file for this site so I'm going to close the issue.

On the point about things being directly related to the PWA, this is an issue with nearly every PWA case study. The great thing about PWAs is that in the process of creating a PWA, organizations often look at their website with a new perspective and decide to address other things as well. 

So the results are often a mixture of improvements. But those improvements would likely never happen with the PWA spurring the efforts.
